prediction_prophet/autonolas/research.py

```diff
@@ -1165,19 +1169,20 @@ def research(
         enc=enc,
     )
 
-    # Spacy loads ~500MB into memory, make it free it with force.
+    # Spacy loads ~500MB into memory, make it free with force.
```

Comment:
I don't think the original is actually a typo 😃. "make it(1) free it(2) with force"
- it(1) is the program
- it(2) is the 500MB memory

Or could be
```suggestion
    # Spacy loads ~500MB into memory. Free it with force.
```

Aaanyway just a nit